utils/spacewalk-common-channels.ini

```diff
@@ -901,6 +901,62 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
 
+[opensuse_leap15_1-aarch64]
```

Comment:
As far as I can understand, since the URLs are different and they don't have a common pattern, we just need different sets of repositories.

Reply:
as long as the channel name and label is different, it is ok.